### PR TITLE
Add debug logs

### PR DESCRIPTION
**Pull request by theofidry**
Small change but I find that useful and in insight would have made it easier for #23

### CODE REVIEW

File: src/BinCommand.php [hunk 1]
```diff
@@ -132,7 +135,7 @@ private function resetComposers(ComposerApplication $application)
     private function chdir($dir)
     {
         chdir($dir);
-        $this->getIO()->writeError('<info>Changed current directory to ' . $dir . '</info>');
+        $this->getIO()->write('<info>Changed current directory to ' . $dir . '</info>', true, IOInterface::VERBOSE);
```
Comment (bamarni) [inline src/BinCommand.php]: Why not leaving it to stderr?

Comment (theofidry) [reply src/BinCommand.php]: it's not an error is it?

Comment (bamarni) [reply src/BinCommand.php]: stderr is not limited to errors, it should also be used for diagnostic / debug output. I think this is also what composer / symfony recommend (cf. https://github.com/symfony/symfony/issues/18744#issuecomment-252883081 for example).

Comment (theofidry) [reply src/BinCommand.php]: Oh I see, well I learned something today! Thanks for the link I'll update the PR shortly

**Pull request merged**